File: MODULE 3/Array/2.c
#include <stdio.h>

int main() {
    int choice;
    float num1, num2, result;
    
    printf("' 1=+ , 2=- , 3=* , 4=/ '\n");
    printf("Enter your choice : ");
    scanf("%d", &choice);
    
    printf("Enter  number 1 : ");
    scanf("%f", &num1);
    
    printf("Enter  number 2 : ");
    scanf("%f", &num2);
    
    switch(choice) {
        case 1:
            result = num1 + num2;
            printf("Result of %.2f + %.2f = %.2f\n", num1, num2, result);
            break;
        case 2:
            result = num1 - num2;
            printf("Result of %.2f - %.2f = %.2f\n", num1, num2, result);
            break;
        case 3:
            result = num1 * num2;
            printf("Result of %.2f * %.2f = %.2f\n", num1, num2, result);
            break;
        case 4:
            if (num2 != 0) 
			{
                result = num1 / num2;
                printf("Result of %.2f / %.2f = %.2f\n", num1, num2, result);
            }
			else
			{
                printf("Error: Division by zero\n");
            }
            break;
        default:
            printf("Invalid choice\n");
    }
    
    return 0;
}

File: Module 7) Python – Collections, functions and Modules/Functions/Lab/2.py
"""• Write a Python program to create a calculator using functions."""

def add():
    sum = num1+num2
    print(sum)

def multi():
    multiplication = num1*num2
    print(multiplication)

def div():
    division = num1/num2
    print(division)

def sub():
    subs = num1-num2
    print(sub)    

menu = ("""
Press 1 for addition
press 2 for Multiplication
press 3 for division
press 4 for substraction""")

print(menu)

choice = int(input("Enter Your choice "))

num1=int(input("Enter no 1 - "))
num2=int(input("Enter no 2 - "))

if choice == 1:
    add()
elif choice == 2:
    multi()
elif choice == 3:
    div()
elif choice == 4:
    sub()
else:
    print("Please enter a right choice ")